apisupport/apisupport.ant/src/org/netbeans/modules/apisupport/project/ui/wizard/NewNbModuleWizardIterator.java

```diff
@@ -161,7 +161,7 @@ public FileObject getCreateProjectFolder() {
         return createdProjectFolder;
     }
     
-    public Set instantiate() throws IOException {
+    public Set<FileObject> instantiate() throws IOException {
```

Comment:
This is an override from an ungenerified superclass. This might break when generics are added to the interface.